Number_types/include/CGAL/FPU.h

```diff
@@ -486,9 +486,15 @@ inline
 FPU_CW_t
 FPU_get_cw (void)
 {
+#ifdef CGAL_ALWAYS_ROUND_TO_NEAREST
+    FPU_CW_t cw;
```

Comment:
To avoid warnings about unused variables when `CGAL_NDEBUG` is defined:
```suggestion
    CGAL_assertion_code(FPU_CW_t cw;)
```

Reply:
Ah, cool -- I didn't notice CGAL_assertion_code.

Thanks. :)